### PR TITLE
Gscond only experiments

### PR DESCRIPTION
**Pull request by frodre**
This PR adds the gscond-only training config and experiments for the microphysics project.  Additionally, for consistency, I update the training data used for all microphysics emulation to v4 (from #1820)

### CODE REVIEW

File: projects/microphysics/train/gscond-only-tscale.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#https://wandb.ai/ai2cm/microphysics-emulation/groups/dqc-precpd-limiter-rnn-limited-qc-87e803-30d-v1-online/workspace?workspace=user-aperkins
```
Comment (nbren12) [inline projects/microphysics/train/gscond-only-tscale.yaml]: note to self: Do we need gscond-only and gscond-tscale...would be nice to choose the better config if possible for now.

File: projects/microphysics/argo/end_to_end.py [hunk 2]
```diff
@@ -251,7 +251,9 @@ def parameters(self):
         prog_config = deepcopy(self.prog_config)
         model_path = os.path.join(model_out_url, "model.tf")
         assert model_path.startswith("gs://")
-        prog_config["zhao_carr_emulation"] = {"model": {"path": model_path}}
+        prog_config["zhao_carr_emulation"] = {
+            "gscond": {"path": model_path, "gscond_cloud_conservative": True}
```
Comment (nbren12) [inline projects/microphysics/argo/end_to_end.py]: note: would be nice to have a `gscond-only.yaml`

File: projects/microphysics/experiments/dense-local-gscond.py [hunk 3]
```diff
@@ -17,9 +17,13 @@
     "rnn-v1-shared-weights": rnn,
 }
 
+# It's a PITA to adjust transforms programmatically at this point time
```
Comment (nbren12) [inline projects/microphysics/experiments/dense-local-gscond.py:20]: :(

File: projects/microphysics/argo/end_to_end.py [hunk 4]
```diff
@@ -248,12 +271,10 @@ def parameters(self):
...
         ml_config["out_url"] = model_out_url
 
-        prog_config = deepcopy(self.prog_config)
         model_path = os.path.join(model_out_url, "model.tf")
-        assert model_path.startswith("gs://")
-        prog_config["zhao_carr_emulation"] = {
-            "gscond": {"path": model_path, "gscond_cloud_conservative": True}
-        }
+        prog_config = set_prognostic_emulation_model(
+            self.prog_config, model_path, gscond_only=True, gscond_conservative=True
```
Comment (nbren12) [inline projects/microphysics/argo/end_to_end.py]: The context isn't showing up in the diff. Just want to ensure this won't apply to gscond+precpd runs.

**Review by nbren12 — APPROVED** [review]
LGTM. Thanks.

File: projects/microphysics/argo/end_to_end.py (hunk 4 above)
Comment (nbren12) [reply projects/microphysics/argo/end_to_end.py]: fixed in a following commit.

File: projects/microphysics/experiments/dense-local-gscond.py (hunk 3 above)
Comment (frodre) [reply projects/microphysics/experiments/dense-local-gscond.py:20]: Haha, this was my concession after starting to write switches to preserve the T-scale vs. no T-scaling case with a single config file😩

**Pull request merged**